search-app/test/cmr/search/test/site/data.clj

```diff
@@ -76,19 +78,30 @@
     (is (= "s3"
            (data/get-short-name coll-data-1)))))
 
-(deftest make-link
+(deftest make-holding-data
   (testing "with an entry title and short name"
-    (is (= {:href "http://cmr.test.host/concepts/C1200000003-PROV1.html"
-            :text "coll3"}
-           (data/make-link cmr-base-url coll-data-1))))
+    (let [data (data/make-holding-data cmr-base-url coll-data-1)]
+      (is (= "http://cmr.test.host/concepts/C1200000003-PROV1.html"
+             (:link-href data)))
+      (is (= "coll3" (:link-text data)))
+      (is (= "s3" (get-in data [:umm "ShortName"])))
+      (is (= "6" (get-in data [:umm "Version"])))))
   (testing "with an entry title, short name, and doi"
-    (is (= {:href "http://dx.doi.org/doi6"
-            :text "coll3"}
-           (data/make-link cmr-base-url coll-data-2)))))
+    (let [data (data/make-holding-data cmr-base-url coll-data-2)]
+      (is (= "http://dx.doi.org/doi6" (:link-href data)))
+      (is (= "coll3" (:link-text data)))
+      (is (= "s3" (get-in data [:umm "ShortName"])))
+      (is (= "7" (get-in data [:umm "Version"]))))))
 
-(deftest make-links
-  (let [coll [coll-data-1 coll-data-2]]
-    (is (= [{:href "http://cmr.test.host/concepts/C1200000003-PROV1.html"
-             :text "coll3"}
-            {:href "http://dx.doi.org/doi6", :text "coll3"}]
-           (data/make-links cmr-base-url coll)))))
+(deftest make-holdings-data
+  (let [coll [coll-data-1 coll-data-2]
```

Comment:
Stylistic nitpick: Probably don't need to introduce this `coll` binding because it's only used in one place (in the next line immediately following this).